uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/sys/PatternFlyClientBundle.java

```diff
@@ -44,4 +44,7 @@
 
     @Source("org/uberfire/client/views/static/jquery-ui/jquery-ui.min.js")
     TextResource jQueryUI();
-}
\ No newline at end of file
+
+    @Source("org/uberfire/client/views/static/monaco-editor/dev/vs/loader.js")
```

Comment:
The MS site suggests to use `dev` for development and `min` for production. 

I don't see any easy way to switch in our build mechanism so wonder if using `min` always would be better?

Our other external libraries use `min` variants.

Reply:
Unfortunately, the `min` version is not working as expected. I reported the issue [here](https://github.com/microsoft/monaco-editor/issues/1608) some days ago. For now, I think we can wait for a response, but if we don't get it fixed from Monaco side, we will need to enable an asset pipeline from our side (we can use this pipeline to minify other assets like `‌daterangepicker.js`). I don't think that it's a good idea to minify Monaco assets from our side now manually, so I'm going to keep this issue on track here: [DROOLS-4734](https://issues.jboss.org/browse/DROOLS-4734).